imageroot/systemd/user/mariadb-app.service

```
@@ -16,12 +18,11 @@ ExecStart=/usr/bin/podman run --conmon-pidfile %t/mariadb-app.pid \
     -v mysql-data:/var/lib/mysql/:Z -v mysql-conf.d:/etc/mysql/conf.d:Z \
     -v sql:/docker-entrypoint-initdb.d:Z \
     -v %S/scripts/create_admin_of_phpmyadmin.sql:/docker-entrypoint-initdb.d/create_admin.sql:Z \
-    -e MARIADB_ROOT_PASSWORD_FILE=/run/secrets/root.secret -e MARIADB_DATABASE=phpmyadmin \
-    -e MARIADB_USER=phpmyadmin -e MARIADB_PASSWORD_FILE=/run/secrets/phpmyadmin.secret \
-    --secret root.secret --secret phpmyadmin.secret \
+    -e MARIADB_ROOT_PASSWORD=${MARIADB_ROOT_PASSWORD} -e MARIADB_DATABASE=phpmyadmin \
+    -e MARIADB_USER=phpmyadmin -e MARIADB_PASSWORD=${MARIADB_PASSWORD} \
```

IMO the long option names are better.
In this case it is possible to use another shortcut: if the `=` is not present, the value is read from the command environment

```suggestion
    --env MARIADB_ROOT_PASSWORD --env MARIADB_DATABASE=phpmyadmin \
    --env MARIADB_USER=phpmyadmin --env MARIADB_PASSWORD \
```

...it is also possible to write `--env MARIADB_*`. All environment variables starting with `MARIADB_` are then imported in the container.

---

the idea is good but the second container does expect the same name vars

https://github.com/NethServer/ns8-mariadb/pull/3/files#diff-02feca4a6bb2d2aaa0d717626e4bd9c5dcbcef8746045252b7f3c8f8d85122e3R18

`    -e PMA_CONTROLPASS=${MARIADB_PASSWORD} \`

I cannot use the shortcut as is 

:-?

---

Yes in that case the shorcut does not work!